src/App.tsx

```diff
@@ -1,26 +1,63 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import logo from "./logo.svg";
+import "./App.css";
+import Footer from "./components/footer";
+import { useTranslation, Trans } from "react-i18next";
+import { useState } from "react";
+import { Suspense } from "react";
+
+const lngs = {
+  en: { nativeName: "English" },
+  pl: { nativeName: "Polish" },
+};
 
 function App() {
+  const { t, i18n } = useTranslation();
+  const [count, setCounter] = useState(0);
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
+        <div>
+          {Object.keys(lngs).map((lng) => (
+            <button
+              key={lng}
+              style={{
+                fontWeight: i18n.resolvedLanguage === lng ? "bold" : "normal",
+              }}
+              type="submit"
+              onClick={() => {
+                i18n.changeLanguage(lng);
+                setCounter(count + 1);
+              }}
+            >
+              {lng}
+            </button>
+          ))}
+        </div>
         <p>
-          Edit <code>src/App.tsx</code> and save to reload.
+          <Trans i18nKey={"description.part1"}>
+            Edit <code>src/App.tsx</code> and save to reload.
+          </Trans>
         </p>
+        <i>{t("counter", { count })}</i>
         <a
           className="App-link"
           href="https://reactjs.org"
           target="_blank"
           rel="noopener noreferrer"
         >
-          Learn React
+          {t("description.part2")}
         </a>
       </header>
+      <Footer t={t} />
     </div>
   );
 }
 
-export default App;
+export default function WrappedApp() {
+  return (
+    <Suspense fallback="...is loading">
+      <App />
+    </Suspense>
```

Comment:
to powinno być owrapowane w pliku index.ts

Reply:
done